packages/now-routing-utils/test/superstatic.spec.js

```diff
@@ -412,12 +412,12 @@ test('convertHeaders', () => {
 
   const expected = [
     {
-      src: '(.*)+/(.*)\\.(eot|otf|ttf|ttc|woff|font\\.css)',
+      src: '^(.*)+(?:\\/(.*))\\.(eot|otf|ttf|ttc|woff|font\\.css)$',
       headers: { 'Access-Control-Allow-Origin': '*' },
       continue: true,
     },
     {
-      src: '404.html',
+      src: '^404\\.html$',
       headers: { 'Cache-Control': 'max-age=300', 'Set-Cookie': 'error=404' },
       continue: true,
     },
```

Comment:
We need a new test to prove that segments in `headers` works properly. You can another route in this test actually.

Reply:
Added another test for segments in the header source